### PR TITLE
Convert setup.py/cfg to pyproject.toml

### PR DESCRIPTION
**Pull request by myakove**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,2 +1,44 @@
...
+authors = [
+    {name = "Meni Yakove", email = "myakove@gmail.com"},
+    {name = "Ruth Netser", email = "rnetser@gmail.com"}
+]
+description = "Wrapper around https://github.com/openshift/openshift-restclient-python"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Operating System :: OS Independent",
+]
+requires-python = ">=3.6"
```
Comment (rnetser) [inline pyproject.toml:19]: 3.9?

File: pyproject.toml [hunk 2]
```diff
@@ -1,2 +1,44 @@
...
+keywords = ["Openshift", "Kubevirt", "CNV"]
+dependencies = [
+    "openshift",
+    "xmltodict",
+    "urllib3",
+    "colorlog",
+    "packaging",
+]
+dynamic = ["version"]
+# ...
```
Comment (rnetser) [inline pyproject.toml:29]: is this needed?

Comment (myakove) [reply pyproject.toml:29]: Yes

File: pyproject.toml (hunk 1 above)
Comment (myakove) [reply pyproject.toml:19]: 3.7, still supported.

**Review by rnetser — APPROVED** [review]

**Comment by myakove** [comment]
!verified

**Pull request merged**